routing/bicycle_directions.cpp

```diff
@@ -366,9 +367,9 @@ void BicycleDirectionsEngine::FillPathSegmentsAndAdjacentEdgesMap(
     if (!segmentRange.IsEmpty())
     {
       auto const it = m_adjacentEdges.find(segmentRange);
-      // A route may be build through intermediate points. So it may contain the same |segmentRange|
+      // A route may be built through intermediate points. So it may contain the same |segmentRange|
       // several times. But in that case |adjacentEdges| corresponds to |segmentRange|
-      // should be the same as well.
+      // should be the same as well except for one case.
```

Comment:
Which case?

Reply:
No such case. This comment is here by mistake. The assert below confirm the fact:
```
      // A route may be built through intermediate points. So it may contain the same |segmentRange|
      // several times. But in that case |adjacentEdges| corresponding to |segmentRange|
      // should be the same.
```